### PR TITLE
LF 9.0.0 - Angular 14

### PR DESCRIPTION
**Pull request by GuillaumeNury**
## :boom: Breaking changes
- Starting from LF9, only one Angular version is supported as it is not possible to support both NG13 and NG14

## :hammer: Technical
- Update Angular from v13 to v14

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1705/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1705/storybook

File: package.json [hunk 1]
```diff
@@ -33,70 +33,67 @@
...
 		"build-compodoc": "compodoc -p ./tsconfig.compodoc-html.json -d compodoc-static",
 		"build-storybook": "ng run storybook:build",
-		"lint": "ng lint"
+		"lint": "ng lint",
+		"lint:scss": "stylelint --config .stylelintrc.json packages/scss/src/**/*.scss"
 	},
 	"dependencies": {
-		"@angular/animations": "13.2.4",
+		"@angular/animations": "14.0.0",
 		"@angular/cdk": "^13.2.4",
```
Comment (lamstutz) [inline package.json:41]: tu mets pas le cdk en 14 ?

**Review by lamstutz — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (GuillaumeNury) [reply package.json:41]: oh c'est un oubli !

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1705/storybook

File: package.json [hunk 2]
```diff
@@ -33,70 +33,67 @@
 		"compodoc": "compodoc -p ./tsconfig.compodoc-json.json -e json -d .storybook",
 		"build-compodoc": "compodoc -p ./tsconfig.compodoc-html.json -d compodoc-static",
 		"build-storybook": "ng run storybook:build",
-		"lint": "ng lint"
+		"lint": "ng lint",
+		"lint:scss": "stylelint --config .stylelintrc.json packages/scss/src/**/*.scss"
```
Comment (shajz) [inline package.json:37]: It this called anywhere? It could replace the gulp command

Comment (GuillaumeNury) [reply package.json:37]: no it's not called anywhere. I dropped `gulp-styleint` in favor of this npm script in order to drop some unmaintained dependencies

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1705/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://recette-auto.lucca.fr/lucca-front-100-54a97c8f-8291-4421-8a36-267a1bdc3c9f/ui-diff.html)

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1705/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1705/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://recette-auto.lucca.fr/lucca-front-124-3277f243-17db-4e27-8a0c-0d93fd18a805/ui-diff.html)

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://recette-auto.lucca.fr/lucca-front-125-268ac05e-7139-4034-93ff-ad35ce30caf0/ui-diff.html)

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1705/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://recette-auto.lucca.fr/lucca-front-127-61cc9e10-9321-434e-bfa1-b46b99d8e99f/ui-diff.html)

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1705/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://recette-auto.lucca.fr/lucca-front-128-e3babe1a-477d-44ae-bafe-826faca3b6c5/ui-diff.html)

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1705/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1705/storybook

**Review by jeremie-lucca — APPROVED** [review]

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://recette-auto.lucca.fr/lucca-front-130-3726b7fe-7f3e-4bf4-aa9c-b77da93d670c/ui-diff.html)

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1705/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://recette-auto.lucca.fr/lucca-front-132-b27433ec-aafa-49e8-b3ae-0a773f86f40c/ui-diff.html)

**Pull request merged**